### PR TITLE
ticdc: Add incremental scan long time region count metrics

### PR DESCRIPTION
**Pull request by hongyunyan**
### What problem does this PR solve?

Issue Number: ref #10862

### What is changed and how it works?
Add incremental scan long time region count metrics.
image

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

### Release note 

```release-note
None
```

### CODE REVIEW

**Comment by asddongmen** [comment]
Please clarify the definition of "long time" within the metric's description.

File: metrics/grafana/ticdc.json [hunk 1]
```diff
@@ -2209,6 +2209,119 @@
             "align": false,
             "alignLevel": null
           }
+        },
+        {
+          "aliasColors": {},
+          "dashLength": 10,
+          "datasource": "${DS_TEST-CLUSTER}",
+          "decimals": 1,
+          "description": "The number of regions that take a long time to scan",
```
Comment (hongyunyan) [inline metrics/grafana/ticdc.json:2218]:
```suggestion
          "description": "The number of regions that take a long time(more than 60s) to scan",
```

**Review by asddongmen — APPROVED** [review]

**Review by lidezhu — APPROVED** [review]

**Pull request merged**